app/scripts/keyring-controller.js

```diff
@@ -173,10 +173,15 @@ module.exports = class KeyringController extends EventEmitter {
     })
   }
 
-  placeSeedWords () {
+  placeSeedWords (cb) {
     const firstKeyring = this.keyrings[0]
     const seedWords = firstKeyring.serialize().mnemonic
     this.configManager.setSeedWords(seedWords)
+
+    if (cb) {
+      cb()
+      this.emit('update')
```

Comment:
> we already transmit an update in the synchronous usage of this method.

where? and if so, then we dont need to do it here at all
emiting updates is unrelated to whether or not the caller wants to be informed when we're done